cfgov/v1/jinja2/v1/includes/article.html

```diff
@@ -55,7 +55,7 @@
         {{ item_introduction.render(data) }}
     </header>
 
-    <div class="post_body">
```

Comment:
Hmm you may want to move this to its own PR. This does cause some minor layout and color shifts. For example, see https://www.consumerfinance.gov/about-us/blog/4-elements-define-personal-financial-well-being/

`post_body` sets the text color to CFPB Dark Gray and adds a top margin. So removing it darkens the text and reduces the spacing.

Before:
<img width="777" alt="Screen Shot 2022-12-23 at 12 34 27 PM" src="https://user-images.githubusercontent.com/704760/209378730-570ecf06-60ac-4200-91f9-8daa4df6754a.png">

After:
<img width="784" alt="Screen Shot 2022-12-23 at 12 34 54 PM" src="https://user-images.githubusercontent.com/704760/209378771-48f9abc0-4ed6-4758-a091-7376501f4a1d.png">

This is so subtle it's probably fine, but there are thousands of pages using this class, so it's probably worth putting this change in its own PR for reference.

Reply:
Yeah, I changed the color here on purpose because I think (although it's within WCAG limits) Dark Gray is harder to read for body text.

Regarding the spacing, this actually is a non-issue, as the `article` template includes a hardcoded `item-introduction` which has a margin-bottom of 3.5em. The margin-top of the post_body in this case always collapses.

Reply:
> Regarding the spacing, this actually is a non-issue, as the article template includes a hardcoded item-introduction which has a margin-bottom of 3.5em. The margin-top of the post_body in this case always collapses.

Isn't that only at the very top of the article, but `post_body` adds margin to the headings contained in the article body? E.g. https://github.com/cfpb/consumerfinance.gov/pull/7429/files#diff-aa89426b239dc3bfc33cc9c93468797377544fe97057c86c82f3ec060082a2deR234

Reply:
Probably still minor, but might be worth having in its own PR since it touches 3k+ pages. Or make this PR more explicitly call out the change in the title, description etc. (If we go looking for the context of the change in the future.)

Reply:
Oh, yeah you're right of course! Good catch! Yeah I'll move this to its own PR, thanks!

Reply:
Removed with justification in https://github.com/cfpb/consumerfinance.gov/pull/7435